src/index.ts

```diff
@@ -167,25 +151,17 @@ function getWebDependencyName(dep: string): string {
   return dep.replace(/\.m?js$/i, '');
 }
 
+interface InstallOptions {
+  hasBrowserlistConfig: boolean;
+  isExplicit: boolean;
+}
+
 export async function install(
   installTargets: InstallTarget[],
-  {
-    isCleanInstall,
-    destLoc,
-    hasBrowserlistConfig,
-    isExplicit,
-    isStrict,
-    isBabel,
-    isOptimized,
-    sourceMap,
-    namedExports,
-    nomodule,
-    nomoduleOutput,
-    externalPackages,
-    dedupe,
-  }: InstallOptions,
```

Comment:
This might be a little controversial, but I just used the exact same schema here as the configuration options. Hopefully now this will be a little less mental overhead between the `install()` function here and the `cli()` function below.

You’ll notice that the 2 parameters `hasBrowserlistConfig` and `isExplicit` that aren’t part of the config are their own options: `InstallOptions`.

Reply:
Just confirming that this means that we're not breaking our existing CLI args? That's fantastic, I don't think the project could pull off a major breaking interface change this early in it's life without getting some pushback (rightfully so!).

Reply:
Nope! We’re not changing any CLI args.

If you notice things like `sourceMap`, I recently realized `yargs` interprets `--no-source-map` the same as `--source-map=false`. But again, that’s exactly how it worked before.

Reply:
fantastic!